src/tools/ceph_objectstore_tool.cc

```diff
@@ -239,12 +232,9 @@ struct pgid_object_list {
         cout << std::endl;
       }
       f->open_array_section("pgid_object");
-      string pgid = i->first.c_str();
-      std::size_t pos = pgid.find("_");
-      if (pos == string::npos)
-        f->dump_string("pgid", pgid);
-      else
-        f->dump_string("pgid", pgid.substr(0, pos));
+      spg_t pgid;
+      if (i->first.is_pg(&pgid))
+        f->dump_string("pgid", stringify(pgid));
```

Comment:
maybe make the field "coll" mandatory and output either "meta" or "0.123" (or whatever), and make the "pg" field only appear if it is actually a pg?  (or not at all?)